packages/spectacle/src/components/code-pane.tsx

```diff
@@ -149,15 +149,13 @@ const CodePane = forwardRef<HTMLDivElement, CodePaneProps>(
        * default theme with no valid values.
        */
       const {
-        size: { width = 1366 },
```

Comment:
🎗 Reminder to self to test this change before merging this PR.

Reply:
I am uncertain what controlling `width` here is trying to achieve (or prevent) so I'm hesitant to want to make this change before knowing what I'm deleting. Sending an empty theme object through (e.g. `theme={{}}`) doesn't appear to affect much change to the codepane.

Maybe you can help me understand the context more fully @gksander ? THANKS!

Reply:
This looks good, as per our discussion.